### DISCUSSION TITLE
How to configure SMTP within the Vaultwarden container?

### DISCUSSION THREAD

**Discussion by hari-bo** [dani-garcia/vaultwarden#2000] [Q&A] [answered]
I've Vaultwarden running without any issues, but if you create an organization you can add new users. Where can i set the SMTP config within this container?

**Comment by BlackDex** [comment, marked as answer]
First, this should not be an issue, but on the discussion page.
Second, read: https://github.com/dani-garcia/vaultwarden/wiki/SMTP-Configuration

**Reply by hari-bo** [reply]
Sorry and thanks for the solution!
